elements/pl-barcode-scan/pl-barcode-scan.mustache

```diff
@@ -0,0 +1,85 @@
+<script>
+    $(() => {
+        $('#pl-barcode-scan-{{uuid}} [data-toggle="popover"]').popover({
+            sanitize: false,
+            container: 'body',
+            template: '<div class="popover pl-barcode-scan-popover" role="tooltip"><div class="arrow"></div><h3 class="popover-header"></h3><div class="popover-body"></div></div>',
+        });
+</script>
+<script>
+    $(() => {
+        // NOTE: Dependency `js-crc` should be loaded in EJS parent view for client-side validation to work
+        const barcode = $('#pl-barcode-scan');
+        const showBadge = (selector) => {
+            $(selector).css('display', 'inline-block');
+        };
+        const hideBadge = (selector) => {
+            $(selector).css('display', 'none');
+        };
+        const toggleInvalidBarcode = (barcode, base36, sha16) => {
+            if (barcode.val().length == 0 || crc16(base36) === sha16) {
+                hideBadge('#pl-barcode-scan-invalid');
+            } else {
+                showBadge('#pl-barcode-scan-invalid');
+            }
+        };
+        const toggleBarcodeRequired = (barcode) => {
+            const barcodeRequired = $('#pl-barcode-scan-required');
+            if (barcodeRequired && barcode.val().length == 0) {
+                showBadge('#pl-barcode-scan-required');
+            } else {
+                hideBadge('#pl-barcode-scan-required');
+            }
+        };
+        const toggleValidBarcode = (barcode, base36, sha16) => {
+            if (crc16(base36) === sha16) {
+                showBadge('#pl-barcode-scan-valid');
+            } else {
+                hideBadge('#pl-barcode-scan-valid');
+            }
+        };
+
+        // listener
+        barcode.keyup((e) => {
+            const sanitized = barcode.val().toLowerCase().trim();
+            const sha16 = sanitized.substring(sanitized.length-4, sanitized.length);
+            const base36 = sanitized.replace(sha16, '');
+
+            toggleBarcodeRequired(barcode);
+            toggleInvalidBarcode(barcode, base36, sha16);
+            toggleValidBarcode(barcode, base36, sha16);
+        });
+    });
+</script>
+
+{{#question}}
+    <!-- Will likely want to add button to add more barcodes that increment up-->
+    <div id="pl-barcode-scan-{{uuid}}" class="d-inline-block form-group">
+        <label for="_pdf_barcode_scan">Barcode</label>
+        <input type="text" id="pl-barcode-scan" style="text-transform:uppercase" name="_pdf_barcode_scan">
```

Comment:
Form-control will make it appear like this: 

![image](https://user-images.githubusercontent.com/7872295/146992884-776ec52a-526a-404a-9f9a-ee79710790f5.png)
![image](https://user-images.githubusercontent.com/7872295/146992935-e84f5d7e-d45a-4d2b-90c4-db41db8c76a6.png)

Instead of this:

![image](https://user-images.githubusercontent.com/7872295/146993020-ea751d78-0bee-4cd6-b256-03b528328863.png)

Which way do you want to go?

Reply:
A combination of the two is ideal - we want the styling of the input to match all other PL (Bootstrap) inputs, but I do like the side-by-side layout of the checkbox.

Reply:
Bump - I'd like this to use Bootstrap styling before I approve.

Reply:
Addressed in 5e22a17.